### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by skyfrench**
Modifications to support:

(in increasing priority order)

- Global styles (e.g. .Input in App.css)
- CSS module files for each class for hard-coded styles (e.g. Input.module.css)
- Styles passed as props

@willrogers - this is not complete (and the tests are not fixed), but opening because of #46.

### CODE REVIEW

**Comment by willrogers** [comment]
Thanks Sky. I realise I'm treading on people's toes a bit by making wide-ranging changes, I don't want to put you off making further contributions. I'm in the unusual situation of having some time to work on this. Would you rather:

* I wait until you've finished everything 
* I finish off work on this branch and fix the tests
* Merge this now and pick up the pieces in further branches

?

I don't mind resolving the conflicts when that comes up, I've got all the VTypes stuff in my head.

**Comment by skyfrench** [comment]
I would like to finish these changes, rather than hand a work in progress over. Perhaps you should merge #46 so that it is not being held up and I'll rebase on top of that and deal with conflicts?

File: src/components/Readback/readback.tsx [hunk 1]
```diff
@@ -9,26 +9,23 @@ import { VType } from "../../vtypes/vtypes";
...
   if (!value) {
     displayedValue = "Waiting for value";
   } else {
     displayedValue = vtypeToString(value, precision);
   }
-
+  style = { backgroundColor: "#383838", color: "#00bb00", ...props.style };
   // Change text color depending on connection state or alarm
   if (!connected) {
     style = {
```
Comment (willrogers) [inline src/components/Readback/readback.tsx]: We can pull the below alarm sensitivity into classes in the module like `.Readback.Major` or something.

**Comment by willrogers** [comment]
This looks nice to me. I thought that the dark was a bit dark and the light was a bit light, so I changed background color to #535353 and #DDDDDD respectively, but background remains white on load and only goes to #DDDDDD on the first toggle. Any ideas?

**Comment by willrogers** [comment]
I'll merge when you're ready.

**Pull request merged**